introduction.md

```diff
@@ -1,3 +1,6 @@
 I am shawn stevens, doing a code school exercise with Github.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?